src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -461,12 +463,15 @@ public override void VisitParameter(IParameterSymbol symbol)
 
             if (includeType)
             {
-                AddRefKindIfRequired(symbol.RefKind);
-
-                if (symbol.IsParams && format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeParamsRefOut))
+                if (format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeParamsRefOut))
                 {
-                    AddKeyword(SyntaxKind.ParamsKeyword);
-                    AddSpace();
+                    AddRefKindIfRequired(symbol.RefKind);
```

Comment:
The behavior is not consistent with VB. In C#, `ref`, `out`, `params` is only included if the type is also included, whereas in VB, `ByRef` and `ParamArray` are included independent of the type. (I've updated the `TestParameterMethodNameTypeModifiers` tests to track that behavior.)

We can change C# to handle `IncludeParamsRefOut` independent of `IncludeType` but that will be a breaking change.